flexmeasures/data/schemas/generic_assets.py

```diff
@@ -31,6 +31,53 @@ def _serialize(self, value, attr, data, **kwargs) -> str:
         return json.dumps(value)
 
 
+class SensorsToShowSchema(ma.SQLAlchemySchema):
+    """
+    Schema for validating the sensors_to_show attribute.
+    """
+
+    def _deserialize(self, value, attr, data, **kwargs) -> list:  # noqa C901
```

Comment:
Actually, you are validating here, which is also useful but not deserializing, which would be to actually unpack any input into a predictable output.

`validate_attributes` seems to use this nicely, but the name is not right.

Reply:
this was addressed i new changes.